static/panes/compiler.ts

```diff
@@ -3461,6 +3485,20 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         });
     }
 
+    setCompilerVersionPopover(version?: CompilerVersionInfo, notification?: string[] | string, compilerId?: string) {
+        if (options.compilerVersionsUrl && compilerId && this.compiler?.name.includes('trunk')) {
```

Comment:
This is a bit hacky -- we should probably have a flag on "trunk" compilers separately. Not all are called "trunk", some are `nightly` (though admittedly those are D or `rust`). Thoughts?

Reply:
I definitely agree, we should probably have a property for it